web-client/src/views/Practitioners/PractitionerCaseIcons.tsx

```diff
@@ -11,8 +11,8 @@ export function PractitionerCaseIcons({
     <div className="multi-filing-type-icon">
       {formattedCase.isSealed && (
         <Icon
-          aria-label={formattedCase.sealedToTooltip}
-          className="sealed-docket-entry"
+          aria-label="sealed"
+          className="sealed-case-entry"
           icon="lock"
           title={formattedCase.sealedToTooltip}
```

Comment:
Just double checking that between the `aria-label` and the `title`, one of the two has a full-text explanation of what is sealed? Want to make sure that we're staying accessible

Reply:
No sir--this is just an oversight. Thank you for catching!

Reply:
The text we want is "sealed"--I was so focused on the aria-label that I just overlooked title entirely. Thanks again :)